src/components/rounds-list/rounds-list.tsx

```diff
@@ -0,0 +1,39 @@
+import { Round } from "@/shared/types/round"
+import {
+  ParticipantImage,
+  ParticipantItem,
+  ParticipantName,
+  ParticipantsList,
+} from "@/components/participants"
```

Comment:
Don't forget to add an export for the components you've created, just to allow us to import using `@/components`.

Reply:
Done!